forms-flow-web/src/components/Form/EditForm/FlowEdit.js

```diff
@@ -75,16 +83,42 @@ const FlowEdit = forwardRef(({ isPublished = false, CategoryType, setWorkflowIsC
   const handleDiscardModal = () => setShowDiscardModal(!showDiscardModal);
   const handleToggleHistoryModal = () => setShowHistoryModal(!showHistoryModal);
 
-  const enableWorkflowChange = ()=>{
+  const enableWorkflowChange = () => {
     setIsWorkflowChanged(true);
     setWorkflowIsChanged(true); // this function passed from parent
   };
 
-  const disableWorkflowChange = ()=>{
+  const disableWorkflowChange = () => {
     setIsWorkflowChanged(false);
     setWorkflowIsChanged(false); // this function passed from parent
   };
 
+  useEffect(() => {
+    if (migration) {
+      setShowMigrationModal(true);
+      setMigration(false);
+    }
+  }, [migration]);
+
+  const handleMigraion = () => {
+    setIsMigraionLoading(true);
+    const migrationData = {
+      mapperId: mapperId,
+      processKey: processData.processKey
+    };
+    processMigrate(migrationData)
+      .then(() => {
+        dispatch(push(`${redirectUrl}formflow`));
```

Comment:
Is this the correct behavior? If someone attempts to migrate within a form, completes the task, and is then redirected to the list page?
@arun-s-aot  @Josephalexantony-aot

Reply:
its safer to route back to listing page as we dont have to handle any data descrepancy issues from edit page. As this is a one time operation per form, only user experience wll be impacted minimally